pyproject.toml

```diff
@@ -40,6 +40,9 @@ azure-identity = {version = "^1.15.0", optional = true}
 azure-keyvault-secrets = {version = "^4.8.0", optional = true}
 google-cloud-kms = {version = "^2.21.3", optional = true}
 resend = {version = "^0.8.0", optional = true}
+opentelemetry-api = "^1.25.0"
+opentelemetry-sdk = "^1.25.0"
+opentelemetry-exporter-otlp = "^1.25.0"
```

Comment:
Otel should not included in lib, but included in proxy docker image, right?

Reply:
yep - add this in the requirements.txt